### PR TITLE
Improve multilingual support and Persian translation

### PR DESCRIPTION
**Pull request by hamidrezakp**
I used `trans` function to replace hard-coded texts in templates and macros.
Persian translation and English texts are okay, but other languages need to write their own translations in `config.toml` file as well.

### CODE REVIEW

**Comment by hamidrezakp** [comment]
OK, now it is ready to be merged.

File: blog/templates/edition-2/page.html [hunk 1]
```diff
@@ -118,12 +121,12 @@ <h2>Other Languages</h2>
...
 
-        <div class="block">
+        <div class="block{% if page.extra.rtl %} left-to-right{% endif %}">
             <h2>About Me</h2>
             <p>
-                I'm a Rust freelancer with a master's degree in computer science. I love systems programming, open source software, and new challenges.
+	        I'm a Rust freelancer with a master's degree in computer science. I love systems programming, open source software, and new challenges.
             </p><p>
-                If you want to work with me, reach out on <a href = "https://www.linkedin.com/in/phil-opp/">LinkedIn</a> or write me at <a href="mailto:job@phil-opp.com">job@phil-opp.com</a>.
+                If you want to work with me, reach out on <a href = \"https://www.linkedin.com/in/phil-opp/\">LinkedIn</a> or write me at <a href=\"mailto:job@phil-opp.com\">job@phil-opp.com</a>.
```
Comment (phil-opp) [inline blog/templates/edition-2/page.html:129]: Why did you escape the `"`s here?

File: blog/templates/edition-2/macros.html [hunk 2]
```diff
@@ -15,11 +15,11 @@
         <h2 class="post-title"><a href="{{ post.path | safe }}">{{ post.title }}</a></h2>
         <div class="post-summary">
             {{ post.summary | safe }}
-            <a class="read-more" href="{{ post.path | safe }}"><em>read&nbsp;more&nbsp;»</em></a>
+	    <a class="read-more" href="{{ post.path | safe }}"><em>{{ trans(key="readmore", lang=lang) | safe }}</em></a>
```
Comment (phil-opp) [inline blog/templates/edition-2/macros.html:18]: Could you undo the whitespace modifications? You added some tab characters for indentation in multiple places, but we mostly use spaces for indentation.

**Review by phil-opp — COMMENTED** [review]
Thank you! I left two small comments inline, otherwise this looks good to me.

File: blog/templates/edition-2/page.html (hunk 1 above)
Comment (hamidrezakp) [reply blog/templates/edition-2/page.html:129]: Oh, i forgot to correct this, this was the text in config file and there it need to be escaped.

File: blog/templates/edition-2/macros.html (hunk 2 above)
Comment (hamidrezakp) [reply blog/templates/edition-2/macros.html:18]: Sure i will check and correct it.

**Comment by phil-opp** [comment]
Thank you!

**Pull request merged**